components/eam/src/physics/crm/rrtmgp/radiation.F90

```diff
@@ -2172,6 +2172,13 @@ subroutine calculate_fluxes_lw(gas_names, gas_vmr, emis_sfc, &
       ncol = size(pmid,1)
       nlev = size(pmid,2)
 
+      ! When lwrad_off is set to true, zero fluxes and return
+      if (lwrad_off) then
+         call reset_fluxes(fluxes_allsky)
+         call reset_fluxes(fluxes_clrsky)
+         return
+      end if
```

Comment:
Don't we need to set qrl and qrlc to zero here like in the other file?

Reply:
qrl and qrlc are computed from fluxes inside calculate_heating_rate(), which are called right after calling the subroutine calculate_fluxes_lw. Since all fluxes are reset to zero inside calculate_fluxes_lw(), qrl and qrlc become zero as well in the case of lwrad_off = True